src/ducks/transactions/actions/AttachedDocsAction/index.js

```diff
@@ -1,10 +1,48 @@
 import React from 'react'
 import flag from 'cozy-flags'
 import { hasBills, hasReimbursements } from 'ducks/transactions/helpers'
+import Bill from 'ducks/transactions/actions/AttachedDocsAction/Bill'
 
 class AttachedDocsAction extends React.PureComponent {
+  renderTransactionRow() {
+    const { transaction } = this.props
+
+    if (hasBills(transaction)) {
+      return this.renderTransactionRowBills()
+    } else if (hasReimbursements(transaction)) {
+      return this.renderTransactionRowReimbursements()
+    } else {
```

Comment:
A transaction cannot have both reimbursements and bills ?

Reply:
Actually I don't have such a case that comes to my mind, but nothing prevents this from happening. So I will handle this case. Thanks.